test/e2e/network/network_policy.go

```diff
@@ -491,6 +493,49 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 			})
 		})
 
+		ginkgo.It("should allow ingress access from namespace on one named port [Feature:NetworkPolicy]", func() {
+			nsBName := f.BaseName + "-b"
+			nsB, err := f.CreateNamespace(nsBName, map[string]string{
+				"ns-name": nsBName,
+			})
+			framework.ExpectNoError(err, "Error creating namespace %v: %v", nsBName, err)
```

Comment:
Shouldn't we clean up the namespace when the test finished?

Reply:
Namespaces are being deleted after test is completed.
[kubernetes/test/e2e/framework/framework.go](https://github.com/kubernetes/kubernetes/blob/a404238d1c90ee00827624249b9223e7644e3f99/test/e2e/framework/framework.go#L404)